src/app.ts

```diff
@@ -13,14 +14,66 @@ interface TemperatureSummary {
   average: number
 }
 
+interface ProcessedReading {
+  city: string
+  time: Date
+  readings: number[]
+}
+
+const processedReadings: ProcessedReading[] = []
+
 export function processReadings(readings: TemperatureReading[]): void {
-  // add here your code
+  readings.forEach((reading: TemperatureReading) => {
+    const city: ProcessedReading | undefined = processedReadings.find(
+      (processedReading: ProcessedReading) =>
```

Comment:
`processedReading: ProcessedReading ` can be inferred since `processedReadings` is a `ProcessedReading[]`